docs/guides/tutorials/trpc.rst

```diff
@@ -0,0 +1,468 @@
+        baseUrl: `http://localhost:${process.env.PORT || 3000}`,
+      });
+
+2. **Create tRPC Context Middleware for Express**
+
+   In **Express**, create middleware to pass the authenticated session and 
+   EdgeDB client to the tRPC context:
+
+   .. code-block:: typescript
+
+      import { type AuthRequest, type Response, type NextFunction } from "express";
+
+      // Middleware to set up tRPC context in Express
+      export const createTRPCContextMiddleware = async (
+        req: AuthRequest,
+        res: Response,
+        next: NextFunction
+      ) => {
+        const session = req.auth?.session(); // Get authenticated session
+        req.context = {
+          session, // Add session to context
+          edgedbClient, // Add EdgeDB client to context
+        };
+        next();
+      };
+
+3. **Set up tRPC Router in Express**
+
+   Use the **tRPC router** in **Express** by including the context middleware 
+   and **EdgeDB Auth** middleware:
+
+   .. code-block:: typescript
+
+      import express from "express";
+      import { appRouter } from "./path-to-router";
+      import { auth } from "./path-to-auth";
+      import { createTRPCContextMiddleware } from "./path-to-context";
+      import { createExpressMiddleware } from "@trpc/server/adapters/express";
+
+      const app = express();
+
+      // EdgeDB Auth middleware to handle sessions
+      app.use(auth.middleware);
+
+      // Custom middleware to pass tRPC context
+      app.use(createTRPCContextMiddleware);
+
+      // tRPC route setup
+      app.use(
+        "/trpc",
+        createExpressMiddleware({
+          router: appRouter,
+          createContext: (req) => req.context, // Use context from middleware
+        })
+      );
+
+      app.listen(4000, () => {
+        console.log('Server running on port 4000');
+      });
+
+4. **Example tRPC Procedure in Express**
+
+   Once the context is set, you can define tRPC procedures that use both the 
+   session and EdgeDB client:
+
+   .. code-block:: typescript
+
+      export const appRouter = t.router({
+        getUserData: t.procedure.query(async ({ ctx }) => {
+          if (!ctx.session) {
+            throw new Error("Not authenticated");
+          }
+          // Query EdgeDB using the client's session
+          const userData = await ctx.edgedbClient.query(`
+            SELECT User { name, email } WHERE .id = <uuid>$id
+          `, { id: ctx.session.user.id });
+
+          return userData;
+        }),
+      });
+
+Conclusion
+----------
+
+By integrating **EdgeDB Auth** into the tRPC context, you ensure that 
+authenticated sessions are securely passed to API procedures, enabling 
+user authentication and protecting routes.
+
+You can also reference these projects for further examples:
+
+- `TurboRepo Template <https://github.com/scotttrinh/turbo-edb-app/blob/main-template/>`_
+- `LookFeel Project <https://github.com/LewTrn/lookfeel>`_ - A real-world 
+  example using **EdgeDB** and **tRPC**.
```

Comment:
Do we want to restate these links here, or just keep them in the beginning or ending?

Reply:
I guess both options make sense: in the beginning for people who'd like to skip to the code, and at the end for a summary.